client/src/components/History/Content/GenericElement.vue

```diff
@@ -1,36 +1,38 @@
+<script setup lang="ts">
+import Vue from "vue";
+import ContentItem from "./ContentItem.vue";
+import { ref } from "vue";
+
+defineProps({
+    dsc: {
+        type: Object,
```

Comment:
Please define more specific types when using typescript.

Reply:
Added some typing annotation for that object.